Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -255,19 +255,17 @@ def sudo_purge
   end
 
   def setup_tar_and_args!
-    # Unset the owner/group for reproducible bottles.
+    # Ensure tar is set up for reproducibility.
```

Comment:
```suggestion
   # Without --only-json-tab bottles are never reproducible
    default_tar_args = ["tar", [].freeze].freeze
    return default_tar_args unless args.only_json_tab?

    # Ensure tar is set up for reproducibility.
```

and use `return default_tar_args` instead of `return ["tar", []].freeze` below